kiosk/src/Components/MainMenu.tsx

```diff
@@ -60,6 +61,17 @@ const MainMenu: React.FC<IProps> = ({ kiosk }) => {
         }
     });
 
+    const onUploadClick = async () => {
+        const sharePointer = await createKioskShareLink({
+            games: kiosk.games,
+        });
+
+        const outputLink = `https://arcade.makecode.com/kiosk?shared=${sharePointer}`;
+
+        alert(`send em to ${outputLink}`);
```

Comment:
Nice. I'm guessing when kiosk starts using react-common we can just take advantage of the share dialog or a simplified version of it.

Reply:
Something like that, I just wanted to implement the smallest amount of ui in this that i could while still having an mvp to keep it simple (same reason this just comes from a button you have to click right now which isn't shippable, instead of hooking it into the gamepad movement flow); for now could just pop to a page with a qr code of it and link saying it's shareable as well if we wanted to add another state.